src/sentry/api/endpoints/organization_details.py

```diff
@@ -171,7 +217,7 @@ def save(self):
             )
         if 'require2FA' in self.init_data and self.init_data['require2FA'] is True:
             org.send_setup_2fa_emails()
-        return org
+        return org, changed_data
 
```

Comment:
This looks great- wondering if there is anything else consuming this that would be expecting just org to be returned but is now receiving a tuple?

Reply:
Grepping over the code base, I don't see any. `organization_index.py` defines its own and no one else imports this serializer.